Eatery/Controllers/BRB/BRBLoginViewController.swift

```diff
@@ -277,6 +277,8 @@ class BRBLoginViewController: UIViewController {
     }
     
     @objc private func loginButtonPressed(_ sender: UIButton) {
+        let payload = BRBLoginPressPayload()
+        AppDevAnalytics.shared.log(payload)
```

Comment:
I would say to do this as `AppDevAnalytics.shared.log(BRBLoginPressPayload())`

Reply:
There are a bunch of cases where this happens elsewhere. I trust you to find them.